src/main/webapp/data/userMapData.json

```diff
@@ -770,6 +786,7 @@
       "St. Gabriels' Secondary",
       "St. Joseph's Institution",
       "St. Margaret's Secondary School",
+      "SUTD",
```

Comment:
This stands for `Singapore University of Technology and Design`, which is an existing entry above.